src/net.cpp

```diff
@@ -643,8 +643,6 @@ CNode* CConnman::ConnectNode(CAddress addrConnect, const char *pszDest, bool fCo
 
 void CNode::CloseSocketDisconnect(CConnman* connman)
 {
-    AssertLockHeld(connman->m_nodes_mutex);
```

Comment:
9cad7f49b732787874162de788f69525df27f030 : same as an already merged https://github.com/dashpay/dash/pull/6319/commits/042e8a4101d8776b5eaf75f559ac705404245205, should rebase and drop this one

Reply:
ooops, I came up to the same fix from 2 different directions